h2o-bindings/bin/custom/python/gen_stackedensemble.py

```diff
@@ -128,11 +129,23 @@ def extend_parms(parms):
 overrides = dict(
     base_models=dict(
         setter="""
-if is_type(base_models, {ptype}):
-    {pname} = [b.model_id for b in {pname}]
+def _get_id(something):
+    if hasattr(something, "model_id"):
+        return something.model_id
+    elif hasattr(something, "grid_id"):
+        return something.grid_id
+    elif is_type(something, str):
+        return something
+    else:
+        raise ValueError("Type {{}} is not supported as a base model.".format(type(something)))
```

Comment:
Same as for R, looks like I forgot to make `H2OGridSearch` a subclass of `Keyed`.
Please make the change (same as for `ModelBase`).